app/views/content/non-uk-teachers/fees-and-funding-for-non-uk-trainees.md

```diff
@@ -75,7 +75,7 @@ As a non-UK candidate, you can expect to pay between £9,250 and about £32,000
 
```

Comment:
I'm not sure why, but that navigation order of this page is different in this PR than in production - it is appearing first on the Non-UK teachers category page?

Reply:
oo I did think it was different, but didn't check 🤔 no idea whats going on there??

Reply:
Both seem to have a nav order of 20.10 - I think we need to resolve this issue separately as it involves reordering all the non-UK pages. I'll raise a separate ticket on this